lib/plugins/aws/deploy/compile/events/cloudWatchLog/index.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const _ = require('lodash');
+
+class AwsCompileCloudWatchLogEvents {
+  constructor(serverless) {
+    this.serverless = serverless;
+    this.provider = this.serverless.getProvider('aws');
+
+    this.hooks = {
+      'deploy:compileEvents': this.compileCloudWatchLogEvents.bind(this),
+    };
+  }
+
+  compileCloudWatchLogEvents() {
+    this.serverless.service.getAllFunctions().forEach((functionName) => {
+      const functionObj = this.serverless.service.getFunction(functionName);
+      let logsNumberInFunction = 0;
+
+      if (functionObj.events) {
+        functionObj.events.forEach(event => {
+          if (event.cloudwatchLog) {
+            logsNumberInFunction++;
+            let LogGroupName;
+            let FilterPattern;
+
+            if (typeof event.cloudwatchLog === 'object') {
+              if (!event.cloudwatchLog.logGroup) {
+                const errorMessage = [
+                  'Missing "logGroup" or "filter" property for logs event ',
```

Comment:
I think it would be better if this said: `Missing "logGroup" property for cloudwatchLog event`. Having `or "filter"` implies that I can specify a `filter` option only which is not true.